src/components/PlexLoginButton/index.tsx

```diff
@@ -38,7 +38,7 @@ const PlexLoginButton: React.FC<PlexLoginButtonProps> = ({
     }
   };
   return (
-    <span className="block w-full rounded-md shadow-sm">
+    <span className="block w-full pb-4 rounded-md shadow-sm">
```

Comment:
This component shouldn't have this padding. It shouldn't be concerned with where or how it's used. Instead, wrap the component with a div where you use it and add the padding there.